english/marshmallow.md

```diff
@@ -562,3 +562,5 @@ and as you do a large pink marshmallow shoots out of your hand into the wall bef
 [Explore your new marshmallow powers.](marshmallow-wizard/marshmallow-wizard.md)
 
 [The evil and heroic marshmallow](the-slayer/marshmallow-slayer.md)
+
+All in a hard days work, do you play some [music](play-music/playmusic.md), or do you do the [robot](happy-robot/happy-robot.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.